test-cases/features/repair-based-operations/longevity-100gb-4h-ReplaceNode.yaml

```diff
@@ -0,0 +1,5 @@
+nemesis_class_name: 'NodeTerminateAndReplace'
+nemesis_interval: 2
+nemesis_filter_seeds: false
```

Comment:
@roydahan Why did you set `nemesis_filter_seeds` to false? "longevity-50GB-3days-authorization-and-tls-ssl.yaml" set 1 seed node only

Reply:
I don't remember, but if it's a problem we can set to true or better increase the number of seeds.

Reply:
Yes, we can't run terminate and replace on seed if one seed in the cluster only.
I created my branch, set seed_num to 3 and run the test with my branch. Can you, please, fix it here

Reply:
you can push to this PR